src/tools/perf/lib/ucp_tests.cc

```diff
@@ -301,6 +311,38 @@ class ucp_perf_test_runner {
         }
     }
 
+    void wait_last_iter(void *buffer, unsigned length)
+    {
+        if ((CMD == UCX_PERF_CMD_PUT) &&
+            (TYPE == UCX_PERF_TEST_TYPE_STREAM_UNI)) {
+            volatile uint8_t *ptr = (volatile uint8_t*)buffer + length - 1;
+            while (*ptr != 1) {
+                progress_responder();
+            }
+        }
+    }
+
+    void reserve_last_iter()
+    {
+        if ((CMD == UCX_PERF_CMD_PUT) &&
+            (TYPE == UCX_PERF_TEST_TYPE_STREAM_UNI)) {
+            m_perf.current.iters++;
+        }
+    }
```

Comment:
why needed?
the last put operation does not have to be counted. we can make it really small (1byte)

Reply:
makes sense to me, done